custom_components/ble_monitor/ble_parser/xiaomi.py

```diff
@@ -563,14 +586,18 @@ def parse_xiaomi(self, data, source_mac, rssi):
             if payload_length < next_start:
                 _LOGGER.debug("Invalid payload data length, payload: %s", payload.hex())
                 break
-            object = payload[payload_start + 3:next_start]
+            object = payload[payload_start + 3 : next_start]
```

Comment:
I think the spaced around : should be removed, I’m unable to test it right now if this also works, normally the format is payload[3:9], isn’t it?

Reply:
Crap.  I didn't notice Python Black ran on save.  I only was purposing the change on line 142.  I'll revert everything else.